### ISSUE TITLE
as a user, I want to be able to manually assign a "consumed on" date

### ISSUE THREAD

**Issue by lungboy** [vinidy/lemon#54]
thinking about data export and daily intake. the date you submit your photo may not be the date you actually consumed the food.

**Comment by MikeBuran** [comment]
Makes sense. Probably should default to current date and time and allow for modification.
